hnsw_redux-python/rustfmt.toml

```diff
@@ -0,0 +1 @@
+max_width = 80
```

Comment:
should this be a workspace level config?

Reply:
Assuming `cargo fmt` picks it up without problem, yes it should. I'll fix it.

Reply:
Fixed (: